scripts/reapply-migrations.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# This script is mostly for deving and quick-rolling migrations/changing +
+# reapplying them
+
+network="$1"
+
+read -r -d '' HASURA_GRANTS <<EOF
+grant connect on database minterop to hasura;
+grant select on all tables in schema public to hasura;
```

Comment:
Where did we end up on the create trigger permissions? Seemed like that caused some issues over the weekend

Reply:
Yep i just copied the script over cause i thought it might be useful down the line, we can update it for sure

Reply:
Importantly for that: They must be executed as the hasura user, but we have the `HASURA_POSTGRES` string in our env for that:

```sql
grant usage on schema hdb_catalog to postgres;
grant select on all tables in schema hdb_catalog to postgres;
grant insert on all tables in schema hdb_catalog to postgres;
```